src/backend/distributed/Makefile

```diff
@@ -74,6 +74,8 @@ $(EXTENSION)--6.0-7.sql: $(EXTENSION)--6.0-6.sql $(EXTENSION)--6.0-6--6.0-7.sql
 	cat $^ > $@
 $(EXTENSION)--6.0-8.sql: $(EXTENSION)--6.0-7.sql $(EXTENSION)--6.0-7--6.0-8.sql
 	cat $^ > $@
+$(EXTENSION)--6.0-9.sql: $(EXTENSION)--6.0-8.sql $(EXTENSION)--6.0-8--6.0-9.sql
```

Comment:
I couldn't find a place to write this, but, it'd be nice if you can update the PR's description so that anyone can follow easier (now or in the future).
I think having a brief description of the new API would be sufficient.